resolwe_bio/processes/variant_calling/wgs_preprocess_bwa2.py

```diff
@@ -0,0 +1,374 @@
+"""Preprocess data for WGS analysis using GATK best practices procedure."""
+import os
+from pathlib import Path
+
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class WgsPreprocess_BWA2(Process):
+    """Prepare analysis ready BAM file.
+
+    This process follows GATK best practices procedure to prepare
+    analysis-ready BAM file. The steps included are read alignment using
+    BWA MEM2, marking of duplicates (Picard MarkDuplicates), BAM sorting,
+    read-group assignment and base quality score recalibration (BQSR).
+    """
+
+    slug = "wgs-preprocess-bwa2"
+    name = "WGS preprocess data"
```

Comment:
Please modify the name so that is will be different from the original BWA mem implementation.